Dockerfile

```diff
@@ -8,8 +8,8 @@ LABEL NAME="official/redmine" \
 # This Dockerfile is based on https://github.com/docker-library/redmine/blob/master/4.0/alpine/Dockerfile
```

Comment:
I know this is just a comment but should the Dockerfile this is based on now be the 4.2/Dockerfile :question:

Reply:
At the time it was created it was based on the 4.0 Dockerfile, I think. Maybe another comparison with the official images was not necessary as the new version run with our docker file natively.